rollup.config.js

```diff
@@ -157,9 +157,9 @@ export const getBabelOptions = ({ useESModules }, targets) => ({
   extensions: babelExtensions,
   runtimeHelpers: true,
   presets: [
+    ['@babel/preset-env', { loose: true, modules: false, targets }],
```

Comment:
rollup disable modules out of the box. You can avoid it.

Reply:
This change is to fix #986. Not sure what you mean. What can I avoid, using `preset-env`?

Reply:
Using `modules: false` in rollup config. Rollup plugin manages modules format out of the box.